tools-local/init-build.proj

```diff
@@ -0,0 +1,323 @@
+      <DummyPackage Include="runtime.win-x86.Microsoft.NETCore.DotnetHostResolver" Version="2.0.0" />
+      <DummyPackage Include="runtime.win-x86.runtime.native.System.Data.SqlClient.sni" Version="4.4.0-preview2-25312-01" />
+      <DummyPackage Include="runtime.win.Microsoft.Win32.Primitives" Version="4.3.0" />
+      <DummyPackage Include="runtime.win.System.Console" Version="4.3.0" />
+      <DummyPackage Include="runtime.win.System.Diagnostics.Debug" Version="4.3.0" />
+      <DummyPackage Include="runtime.win.System.IO.FileSystem" Version="4.3.0" />
+      <DummyPackage Include="runtime.win.System.Net.Primitives" Version="4.3.0" />
+      <DummyPackage Include="runtime.win.System.Net.Sockets" Version="4.3.0" />
+      <DummyPackage Include="runtime.win.System.Runtime.Extensions" Version="4.3.0" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.App" Version="2.0.0-beta-001737-00" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.DotNetHost" Version="1.0.1" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.DotNetHost" Version="1.1.0" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.DotNetHostPolicy" Version="1.0.5" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.DotNetHostPolicy" Version="1.1.0" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.DotNetHostResolver" Version="1.0.1" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.DotNetHostResolver" Version="1.1.0" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.Jit" Version="1.0.7" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.Jit" Version="1.1.0" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR" Version="1.0.7" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR" Version="1.1.0" />
+      <DummyPackage Include="runtime.win7-x64.Microsoft.NETCore.Windows.ApiSets" Version="1.0.1" />
+      <DummyPackage Include="runtime.win7-x64.runtime.native.System.IO.Compression" Version="4.3.0" />
+      <DummyPackage Include="runtime.win7.System.Private.Uri" Version="4.3.0" />
+      <DummyPackage Include="xunit" Version="2.2.0" />
+      <DummyPackage Include="xunit.abstractions" Version="2.0.1" />
+      <DummyPackage Include="xunit.assert" Version="2.2.0" />
+      <DummyPackage Include="xunit.core" Version="2.2.0" />
+      <DummyPackage Include="xunit.extensibility.core" Version="2.2.0" />
+      <DummyPackage Include="xunit.extensibility.execution" Version="2.2.0" />
+      <DummyPackage Include="xunit.runner.visualstudio" Version="2.2.0" />
+    </ItemGroup>
+
+    <BuildDummyPackages Packages="@(DummyPackage)"
+                        OutputPath="$(SourceBuiltPackagesPath)" />
+  </Target>
+
+  <Target Name="BuildReferenceAssemblies">
+    <Exec Command="$(DotNetCliToolDir)/dotnet build $(SubmoduleDirectory)/reference-assemblies/build.proj" />
```

Comment:
Since we are using `$(DotNetCliToolDir)/dotnet` to build this project, can these just be `<MSBuild` tasks?  Same for the ones above.

Reply:
I didn't attempt to change these but I suspect we will have the same issue I had with the task project build where it cannot successfully compile when the root is building via `dotnet msbuild.dll` as opposed to `dotnet msbuild`

Reply:
oh right.  good point.